webapp/src/reducers/index.js

```diff
@@ -160,4 +178,6 @@ export default combineReducers({
     gitlabUsers,
     rhsPluginAction,
     rhsState,
+    yourPrsDetails,
+    reviewsDetails,
```

Comment:
```suggestion
    reviewDetails,
```

why this is repeated/still not fixed? I have already given this comment  that if you use `details` then 1st word must be singular like `reviewDetails`